public/tier1/UtlStringMap.h

```diff
@@ -55,9 +55,9 @@ class CUtlStringMap
 		return m_SymbolTable.Find( pString );
 	}
 
-	UtlSymId_t AddString( const char *pString )
+	UtlSymId_t AddString( const char *pString, bool &result )

```

Comment:
Also why did you used the different name and type in here for the new argument? Is it different to other function prototypes you've got in the pr?

Reply:
My bad, the mass replacement didn't quite cover everything.